config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/loader/ListenPortLoader.java

```diff
@@ -35,7 +39,13 @@ public void load(final Bundle bundle, final Element element) {
         final Set<String> enabledFeatures = getChildElementsTextContents(getSingleChildElement(listenPortElement, ENABLED_FEATURES), STRING_VALUE);
         final String targetServiceReference = getSingleChildElementTextContent(listenPortElement, TARGET_SERVICE_REFERENCE);
         final ListenPortTlsSettings tlsSettings = getTlsSettings(listenPortElement);
-        final Map<String, Object> properties = mapPropertiesElements(getSingleChildElement(listenPortElement, PROPERTIES, true), PROPERTIES);
+        Map<String, Object> properties = mapPropertiesElements(getSingleChildElement(listenPortElement, PROPERTIES, true), PROPERTIES);
+        if (!equalsAny(protocol, PROTOCOL_FTP, PROTOCOL_FTPS)) {
```

Comment:
What are the intended values for protocol?

Reply:
can be HTTP, HTTPS, SSH, FTP or FTPS
but in this case I am removing property that does not make sense if not FTP(s)